static/usr/lib/core20/run-snapd-from-snap

```diff
@@ -7,7 +7,8 @@ set -eux
 
 # run_on_unseeded will mount/run snapd on an unseeded system
 run_on_unseeded() {
-    SEED_SNAPD="$(find /var/lib/snapd/seed/snaps/ -name "snapd_*.snap")"
+    # XXX: read the snapd snap to bootstrap from via /var/lib/snapd/modeenv
+    SEED_SNAPD="$(find /var/lib/snapd/seed/ -name "snapd_*.snap")"
```

Comment:
couldn't/would't snapd be already mounted via snap-bootstrap ?

Reply:
@pedronis at the moment snap-bootstrap does not mount snapd. Imho, it should be validating and mounting snapd. As otherwise, one can freeze execution, and inject unverified snapd before this line of code runs.